README.md

```diff
@@ -57,6 +59,8 @@ This project was created using ASP.NET Core 6.0. This application aims to demons
 
     ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
 
+> **NOTE:** By default, the dashboard listing is hidden in the sample. If you want to render a dashboard with a specific listing of dashboards, you need to add a suburl to the web address and retrieve the dashboard listing accordingly.(e.g., https://localhost:5001/dashboardlisting).
```

Comment:
We represent the dashboard embedding by default without the dashboards listing sidebar. You must navigate to the "dashboardlisting" URL ( such as https://localhost:5001/dashboardlisting) to enable the dashboards list.

Reply:
Content updated